packages/frontapp/src/app/components/BackButton/index.tsx

```diff
@@ -21,7 +22,9 @@ export const GoBack = styled.a`
 const BackButton = () => {
   const navigate = useNavigate();
   const handleGoBack = () => {
-    navigate(-1);
+    window.history.state.idx === 0
```

Comment:
can you please explain what is `state.idx`

Reply:
State idx always return a number, when you visit the page for the first time it returns 0. When you go to another page state idx will be incrmented. 
I checked, if we are visiting the page for the first time.

Also, we don't have to include state.idx === 0, as if state.idx equal to zero it's false by default )